en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -0,0 +1,212 @@
+# Publishing a sample API to Service Catalog
```

Comment:
```suggestion
# Exposing an Integration Service as a Managed API
```
This is the use case of the 'service catalog'. Therefore, shall we change the title to this
@samgnaniah

Reply:
+1.

Reply:
Changed in a0c600c